src/sporemodder/view/syntax/XmlSyntax.java

```diff
@@ -101,7 +101,10 @@ public class XmlSyntax implements SyntaxFormatFactory {
 	public boolean isSupportedFile(File file) {
 		try {
 			// Unfortunately, this syntax highlighter is too slow for "big" files
-			return file.getName().endsWith(".xml") && Files.size(file.toPath()) < 10240;
+			return Files.size(file.toPath()) < 10240 && (
+				file.getName().endsWith(".xml") |
+				file.getName().endsWith(".eapdPixie")
+			);
```

Comment:
I think it's better to revert the conditions. Checking file size might be slower, no need to do it if it is not one of the extensions.